suite/tests/CMakeLists.txt

```diff
@@ -2697,6 +2697,12 @@ endif ()
         set(tool.drcacheoff.legacy_rawtemp ON) # no preprocessor
       endif ()
 
+      # Test reading a trace in sharded snappy-compressed files.
+      if (libsnappy)
+        torunonly_drcacheoff(snappy ${ci_shared_app} ""
+          "@-indir@${PROJECT_SOURCE_DIR}/clients/drcachesim/tests/drmemtrace.chase-snappy.x64.tracedir@-simulator_type@basic_counts" "")
```

Comment:
torunonly_drcacheoff goes and runs the app and generates a trace.  If there's no support for *producing* snappy-compressed files, and we can only test reading a checked-in trace, this should look more like the tool.drcacheoff.legacy test which uses torunonly_api to directly invoke drcachesim on the trace file: no need to go run an app and generate a trace that is then never looked at.

Reply:
Ah, makes sense.